src/docs/plugins/serverless.md

```diff
@@ -355,13 +356,33 @@ permalink:
 
 Eleventy fetches the value stored at `eleventy.serverless.path.id` (using [lodash get](https://lodash.com/docs/4.17.15#get)) and does an additional get on the pagination data in `authors`.
 
-For example:
+For example given this `authors` data :
+
+```json
+{
+  "zachleat": {
+    "name": "Zach Leatherman",
+    "content": "Content for Zach Leatherman"
+  },
+  "brianjon": {
+    "name": "Brian Jones",
+    "content": "Content for Brian Jones"
+  },
+  "janedanc": {
+    "name": "Jane Dance",
+    "content": "Content for Jane Dance"
+  }
+}
+```
 
 1. A request is made to `/authors/zachleat/`
-1. The dynamic URL slug for the `possum` serverless function `/authors/:id/` matches `zachleat` to `:id`. This sets `"zachleat"` in the `eleventy.serverless.path.id` Global Data.
-1. Because `pagination.serverless` has the value `"eleventy.serverless.path.id"`, we use lodash.get to select the key `"zachleat"` from Global Data.
-1. An additional `lodash.get(authors, "zachleat")` returns a single chunk of data for one author.
-1. Pagination only operates on that one selected page for rendering.
+1. The dynamic URL slug for the `possum` serverless function `/authors/:id/` matches `zachleat` to `:id`. 
+1. This in turn sets `eleventy.serverless.path.id` in your Global Data to `"zachleat"`.
+1. As `pagination.serverless` is set to `"eleventy.serverless.path.id"` we can use it as a key via `lodash.get(authors, "zachleat")` to return data a chunk of data from `authors`. 
+1. Via `pagination.alias`, your template now has access to `author.name` and `author.content` for `"zachleat"`
+1. And as `pagination.size` is set to 1, the pagination only operates on that one selected page for rendering.
+
+{% callout "info", "md" %}You have to use an object for your pagination data. If you use an array you will get unexpected results.{% endcallout %}
```

Comment:
Is this line true? You can’t use `/authors/0/` to fetch `[{ name: "", … }]`?

Reply:
I think so. 

If your posts object is an array, you _can_ use `/authors/0/` to fetch the first post, but that probably isn't very helpful. 

You almost certainly want to match a serverless url against a full slug name and not just a numeric value. 

Working repo here:
https://github.com/dwkns/eleventy-serverless-demo